editor/docks/filesystem_dock.cpp

```diff
@@ -775,13 +775,13 @@ void FileSystemDock::_navigate_to_path(const String &p_path, bool p_select_in_fa
 			item = item->get_next();
 		}
 		if (p_grab_focus) {
-			tree->grab_focus();
+			tree->grab_focus(true);
```

Comment:
If the intention is to hide focus everywhere, you can make `true` the default argument of C++ method, but still bind it with `false` default.

Reply:
It isn't. There are places where showing focus is desirable.